src/components/overflow-menu/overflow-menu.js

```diff
@@ -63,17 +64,21 @@ export const getMenuOffset = (menuBody, direction) => {
   }
 
   if (componentsX) {
+    const flip = menuBody.classList.contains(`${settings.prefix}--overflow-menu--flip`);
```

Comment:
is this class identical to `this.options.classMenuFlip`?

Reply:
Great catch @emyarod! - Fixed.